private/rules/maven_publish.bzl

```diff
@@ -104,6 +109,8 @@ When signing with GPG, the current default key is used.
         "source_jar": attr.label(
             allow_single_file = True,
         ),
+        "extra_classifiers": attr.string_list(),
```

Comment:
If you follow the suggestion above, this either becomes a `string_list`, or (if you'd like to wait as long as possible before preparing the command line) a `string_dict`

Reply:
I would need string_label_dict which is not supported yet on attr. https://stackoverflow.com/questions/43396925/emulate-string-to-label-dict

Reply:
Maybe can flip the keys/values and use [label_keyed_string_dict](https://bazel.build/rules/lib/attr#label_keyed_string_dict)?